xonsh/xoreutils/util.py

```diff
@@ -0,0 +1,15 @@
+def arg_handler(args, out, short, key, val, long=None):
```

Comment:
Maybe call this file `tools.py` to be consistent with the `xonsh/tools.py`. This is not a blocker, though.